lab_4_retrieval_w_clustering/main.py

```diff
@@ -26,6 +37,10 @@ def get_paragraphs(text: str) -> list[str]:
     Returns:
         list[str]: Paragraphs from document.
     """
+    if not isinstance(text, str) or not text:
+        raise ValueError('Invalid arguments in get_paragraphs')
```

Comment:
of couse you can explain what is valid arguments means

Reply:
explained